CalibrationWidgets/measureMachinePopup.py

```diff
@@ -3,14 +3,14 @@
 This allows the user interact with the z-axis when it is the content of a popup
 
 '''
-from   kivy.uix.gridlayout                       import   GridLayout
-from   kivy.properties                           import   ObjectProperty
-from   kivy.properties                           import   StringProperty
-from   UIElements.touchNumberInput               import   TouchNumberInput
-from   UIElements.triangularCalibration          import   TriangularCalibration
-from   UIElements.adjustZCalibrationDepth        import   AdjustZCalibrationDepth
-from   UIElements.measureDistBetweenMotors       import   MeasureDistBetweenMotors
-from   kivy.uix.popup                            import   Popup
+from   kivy.uix.gridlayout								import   GridLayout
+from   kivy.properties									import   ObjectProperty
+from   kivy.properties									import   StringProperty
```

Comment:
I've been switching from spaces to tabs for the spacing on these because I think it's cleaner. It renders right in my text editor but wrong here. Do the tabs look OK in other text editors?